category_theory/limits/binary_products.lean

```diff
@@ -0,0 +1,81 @@
+-- Copyright (c) 2018 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Scott Morrison
+
+import category_theory.limits.products
+
+universes u v
+
+open category_theory
+
+namespace category_theory.limits
+
+inductive two : Type v
```

Comment:
`@[derive decidable_eq]`

Reply:
Added. (Also on the inductive types introduced in `equalizer.lean` and `pullback.lean`.